forloop_modules/globals/scraping_utilities_handler.py

```diff
@@ -219,10 +236,26 @@ def get_browser_view_elements(self):
 
     def get_browser_view_selected_elements(self):
         return self.browser_view_selected_elements
+    
+    def get_group_idex_of_webpage_element(self, element):
+        element_group_index = None
+        
+        for i, group in enumerate(suh.browser_view_selected_element_groups):
```

Comment:
This looks weird, what about some clever list comprehension? Maybe something like this?
[(element in group) for group in suh.browser_view_selected_element_groups].index(True)